code/iexfeed/iex.q

```diff
@@ -35,7 +37,7 @@ get_last_trade:{tab:{[syms]
    / Construct the GET request
    suffix:.iex.trade_suffix[syms];
    / Parse json response and put into table. Trade data from https://iextrading.com/developer/
-   data:.j.k .iex.get_data[.iex.main_url;suffix];
+   data:.j.k .iex.get_data[suffix];   
```

Comment:
Can you remove the trailing whitespace here?

Reply:
Done